lib/auth/index.js

```diff
@@ -19,14 +19,14 @@ const getAuth = options =>
     responseType: 'token id_token',
     scope: 'openid profile',
     redirectUri: `${getBaseUrl()}/auth/signed-in`,
```

Comment:
Ser at det er konfigurert opp feil i auth0, og login i test fungerer ikke. Auth0 forventer `https://www.test.digitallibrary.io/auth/signed-in` ikke `https://test.digitallibrary.io/auth/signed-in`. Hør evt. med @gunnarvelle hva dere bør gjøre.

Reply:
Dette må fikses utenfor frontenden.